pkg/interfaces/contracts/pool-utils/IManagedPool.sol

```diff
@@ -0,0 +1,316 @@
+     */
+
+    function setMustAllowlistLPs(bool mustAllowlistLPs) external;
+
+    /**
+     * @notice Adds an address to the LP allowlist.
+     * @dev Will fail if the address is already allowlisted.
+     * Emits the AllowlistAddressAdded event. This is a permissioned function.
+     * @param member - The address to be added to the allowlist.
+     */
+    function addAllowedAddress(address member) external;
+
+    /**
+     * @notice Removes an address from the LP allowlist.
+     * @dev Will fail if the address was not previously allowlisted.
+     * Emits the AllowlistAddressRemoved event. This is a permissioned function.
+     * @param member - The address to be removed from the allowlist.
+     */
+
+    function removeAllowedAddress(address member) external;
+
+    /**
+     * @notice Returns whether the allowlist for LPs is enabled.
+     */
+    function getMustAllowlistLPs() external view returns (bool);
+
+    /**
+     * @notice Check whether an LP address is on the allowlist.
+     * @dev This simply checks the list, regardless of whether the allowlist feature is enabled.
+     * @param member - The address to check against the allowlist.
+     * @return true if the given address is on the allowlist.
+     */
+    function isAddressOnAllowlist(address member) external view returns (bool);
+
+    // Management fees
+
+    /**
+     * @notice Collect any accrued AUM fees and send them to the pool manager.
+     * @dev This can be called by anyone to collect accrued AUM fees - and will be called automatically on
+     * joins and exits.
+     * @return The amount of BPT minted to the manager.
+     */
+    function collectAumManagementFees() external returns (uint256);
+
+    /**
+     * @notice Setter for the yearly percentage AUM management fee, which is payable to the pool manager.
+     * @dev Attempting to collect AUM fees in excess of the maximum permitted percentage will revert.
+     * To avoid retroactive fee increases, we force collection at the current fee percentage before processing
+     * the update. Emits the ManagementAumFeePercentageChanged event. This is a permissioned function.
+     * @param managementAumFeePercentage - The new management AUM fee percentage.
+     * @return amount - The amount of BPT minted to the manager before the update, if any.
+     */
+    function setManagementAumFeePercentage(uint256 managementAumFeePercentage) external returns (uint256);
+
+    /**
+     * @notice Returns the management AUM fee percentage as an 18-decimal fixed point number and the timestamp of the
+     * last collection of AUM fees.
+     */
+    function getManagementAumFeeParams()
+        external
+        view
+        returns (uint256 aumFeePercentage, uint256 lastCollectionTimestamp);
+
+    // Circuit Breakers
+
+    /**
+     * @notice Set a circuit breaker for one or more tokens.
+     * @dev This is a permissioned function, and disabled if the pool is paused. The lower and upper bounds
```

Comment:
```suggestion
     * @dev This is a permissioned function. The lower and upper bounds
```
It is disabled when the pool is paused, like all current permissioned functions, but we don't say this in any of the other permissioned function comments. (Should be all or none, to avoid the impression that this one is different.)